tracer/contrib/database/sqltraced/utils.go

```diff
@@ -0,0 +1,136 @@
+package sqltraced
+
+import (
+	"database/sql/driver"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/DataDog/dd-trace-go/tracer"
+	"github.com/go-sql-driver/mysql"
```

Comment:
these imports are problematic. this means that somebody using mysql needs to install postgres and vice versa. these could have C dependencies which might require the database to be installed, etc. in general, users of this package shoudn't require any unneeded dependencies.